src/plugins/intel_cpu/tests/functional/subgraph_tests/src/matmul_weights_decompression.cpp

```diff
@@ -162,11 +186,14 @@ class MatmulWeightsDecompression : public testing::WithParamInterface<MatmulWeig
         auto scaleshift_const_shape = scaleshift_target_shape;
         if (reshape_on_decompression_constant)
             scaleshift_const_shape.erase(std::remove(scaleshift_const_shape.begin(), scaleshift_const_shape.end(), 1), scaleshift_const_shape.end());
-        if (add_subtract) {
-            auto shift_const = ngraph::builder::makeConstant<uint8_t>(weights_precision, scaleshift_const_shape, {}, true, 7);
+        if (decompression_subtract_type != DecompressionSubtractType::empty) {
+            auto subtract_shape = decompression_subtract_type == DecompressionSubtractType::full ? scaleshift_const_shape : Shape({});
+            auto shift_const = ngraph::builder::makeConstant<uint8_t>(weights_precision, subtract_shape, {}, true, 7);
```

Comment:
Should we create `shift_const` in the same way like `weights`?
```suggestion
            auto shift_const = ngraph::builder::makeConstant<int8_t>(weights_precision, subtract_shape, {}, true, up_to);
```

Reply:
Good point. Added